src/config.cc

```diff
@@ -523,17 +523,21 @@ Status Config::parseConfigFromString(std::string input, int line_number) {
   if (kv.size() != 2) return Status(Status::NotOK, "wrong number of arguments");
   if (kv[1] == "\"\"") return Status::OK();
 
-  kv[0] = Util::ToLower(kv[0]);
-  auto iter = fields_.find(kv[0]);
+  std::string configKey = Util::ToLower(kv[0]);
+  const char ns_str[] = "namespace.";
+  size_t ns_str_size = sizeof(ns_str) - 1;
+  if (!strncasecmp(kv[0].data(), ns_str, ns_str_size)) {
+      // namespace should keep key case-sensitive
+      configKey = kv[0];
+      tokens[kv[1]] = kv[0].substr(ns_str_size);
+  }
+  auto iter = fields_.find(configKey);
```

Comment:
```suggestion
  std::string field_key = Util::ToLower(kv[0]);
  const char ns_str[] = "namespace.";
  size_t ns_str_size = sizeof(ns_str) - 1;
  if (!strncasecmp(kv[0].data(), ns_str, ns_str_size)) {
      // namespace should keep key case-sensitive
      field_key = kv[0];
      tokens[kv[1]] = kv[0].substr(ns_str_size);
  }
  auto iter = fields_.find(field_key);
```

Reply:
The variable name should be spliced by `_` instead of CamelCase. Sorry that we didn't claim the code style clearly in our contribution guide.